task/check-noarch.yaml

```diff
@@ -52,31 +52,9 @@ spec:
     - name: check-noarch
       image: $(params.script-environment-image)
```

Comment:
For the CI purposes, please temporarily bump the script-environment-image reference to the image that has been built in https://github.com/konflux-ci/rpmbuild-pipeline-environment-container/pull/26